_common.scss

```diff
@@ -13,4 +13,8 @@
     .mat-chip .mat-chip-avatar {
         font-size: 0.625rem;
     }
+
+    .mat-button-toggle .mat-button-toggle-label-content {
+        line-height: 36px;
+    }
```

Comment:
Not sure what the final result should be here. The button itself is 36px now, but with the border it's 38px. @huayunh what's the expected style here? Should the entire container, including the border be 36 or is it good to go as is?
![Screen Shot 2021-09-23 at 8 45 59 AM](https://user-images.githubusercontent.com/13989985/134509211-5398c298-e407-4788-8682-20be519f716e.png)
![Screen Shot 2021-09-23 at 8 45 48 AM](https://user-images.githubusercontent.com/13989985/134509214-ceaebdf9-71a2-415f-b759-518ef5d76142.png)
.

Reply:
36px.

Theoretically, it should be equally tall as a regular button, outlined or filled.